File: app/src/main/java/io/github/kutaiba_nezar_kashmar/multiverse_of_geeks/domain/response/CastResponse.java
package io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.domain.response;

import java.util.ArrayList;

import io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.domain.Cast;

public class CastResponse
{
  private int id;
  private int gender;
  private String known_for_department;
  private String name;
  private String profile_path;
  private String character;
  private ArrayList<Cast> cast = null;

  public Cast getCast()
  {
    return new Cast(id, gender, known_for_department, name, profile_path,
        character);
  }

  public ArrayList<Cast> getCastList()
  {
    return cast;
  }
}

File: app/src/main/java/io/github/kutaiba_nezar_kashmar/multiverse_of_geeks/model/domain/response/CastResponse.kt
package io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.response

import io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.local.Cast
import java.util.ArrayList

class CastResponse
{
    val id: Int = 0
    val gender: Int = 0
    val known_for_department: String = ""
    val name: String = ""
    val profile_path: String = ""
    val character: String = ""
    val cast: List<Cast> = ArrayList()

    fun getCast(): Cast
    {
        return Cast(
            id, gender, known_for_department, name, profile_path, character
        )
    }

    fun getCastList() : List<Cast>
    {
        return cast
    }
}
